aws-datastore/src/androidTest/java/com/amplifyframework/datastore/MultiAuthSyncEngineInstrumentationTest.java

```diff
@@ -807,12 +810,18 @@ private void configure(Class<? extends Model> modelType,
 
         // Setup DataStore
         DataStoreConfiguration dsConfig = DataStoreConfiguration.builder()
-                                                                .errorHandler(exception -> {
-                                                                    Log.e(tag,
-                                                                          "DataStore error handler received an error.",
-                                                                          exception);
-                                                                })
-                                                                .build();
+                                                .errorHandler(exception -> {
+                                                    Log.e(tag,
+                                                          "DataStore error handler received an error.",
+                                                          exception);
+                                                })
+                                                .syncExpression(modelSchema.getName(), new DataStoreSyncExpression() {
+                                                    @Override
+                                                    public QueryPredicate resolvePredicate() {
+                                                        return Where.id("FAKE_ID").getQueryPredicate();
+                                                    }
+                                                })
```

Comment:
can't we use lambda here?
```
.syncExpression(modelSchema.getName(), () -> Where.id("FAKE_ID").getQueryPredicate())
```

Reply:
`Where.id()` helper method also has a bug with ambiguous column names IIRC. I guess it's fine unless we start writing multiauth tests that has model relationship

Reply:
Yeah...I think for the purposes of this test, we're using a single table for each test case so we should be OK